api/queries/community/threadConnection.js

```diff
@@ -10,9 +10,15 @@ import {
 import { getThreadsByChannels } from '../../models/thread';
 import { canViewCommunity } from '../../utils/permissions';
 
+export type CommunityThreadConnectionPaginationOptions = {
+  after: string,
+  first: number,
+  sort: 'NEW' | 'TRENDING',
```

Comment:
Above we had 'new' and 'trending' but here they are all caps. Should we be consistent with these?

Reply:
No I don't think so? One is the GraphQL stuff (which has to be all caps) and one is internally the argument for the db query. (which we never do all caps)

Reply:
Referring to this: https://github.com/withspectrum/spectrum/pull/3798/files#diff-8e46094eed25a21b0755138dc52cf17eR77

No worries either way